src/components/VmActions/index.js

```diff
@@ -134,15 +136,40 @@ class VmActions extends React.Component {
 
     const vncConsole = vm.get('consoles').find(c => c.get('protocol') === 'vnc')
     const hasRdp = isWindows(vm.getIn(['os', 'type']))
-    const consoles = vm.get('consoles').map((c) => ({
-      priority: 0,
-      shortTitle: msg[c.get('protocol') + 'Console'](),
-      icon: <Icon name='external-link' />,
-      id: `${idPrefix}-button-console-${c.get('protocol')}`,
-      confirmation: (
-        <ConsoleConfirmationModal consoleId={c.get('id')} vm={vm} />
-      ),
-    })).toJS()
+    let consoles = []
+    if (isOnCard) {
```

Comment:
`isCard` was already used before to separate card-scenario from toolbar scenario - however the `getDeafaultAction` step was common to both paths. Right know we no longer have a clean point where the scenarios diverge. Perhaps it's time to break this method into smaller ones i.e. `getDefaultActionsForCard()` and `getDefaultActionsForToolbar()` ?

Reply:
IMHO most of the logic is shared and both of the options using it, I think that we should not separate it,
but if most of the team thinks different I'll change it.

Reply:
@bamsalem @rszwajko I don't have a strong opinion on this. The only difference is the consoles menu so not sure it worth separating in current phase.

Reply:
ok. let's leave it as it is.